decidim-core/config/initializers/new_framework_defaults_7_0.rb

```diff
@@ -109,7 +109,7 @@
 # If you are upgrading and have not set `cookies_serializer` previously, your cookie serializer
 # is `:marshal`. The default for new apps is `:json`.
 #
-# Rails.application.config.action_dispatch.cookies_serializer = :json
+Rails.application.config.action_dispatch.cookies_serializer = :json
```

Comment:
I think we should first do the hybrid option (see line 115)

Reply:
Yeah ... I already have it prepared.

Reply:
Done in [2a9af45](https://github.com/decidim/decidim/pull/14735/commits/2a9af458f3fe3e336d621bfb604d922e080a7170)

Reply:
We discussed internally to avoid :marshal serialization in cookies